lua/acf/damage/damage_cl.lua

```diff
@@ -44,39 +54,46 @@ local function Add(Entity)
 		hook.Add("PostDrawOpaqueRenderables", "ACF_RenderDamage", RenderDamage)
 	end
 
-	Damaged[Entity] = true
+	Damaged[Entity] = Entity:GetTable()
 
 	Entity:CallOnRemove("ACF_RenderDamage", function()
 		Remove(Entity)
 	end)
 end
 
-Network.CreateReceiver("ACF_Damage", function(Data)
-	for Index, Percent in pairs(Data) do
-		local Entity = ents.GetByIndex(Index)
+do
```

Comment:
Same deal here, just plopped it in a `do/end` with some small localized functions.